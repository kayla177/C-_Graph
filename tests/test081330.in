ENTITY MRRX4FAW name_MRRX institution
ENTITY LMWWZQST name_LMWW journal
ENTITY SFLY3MWC name_SFLY conference
ENTITY UT3LHDM9 name_UT3L author
ENTITY 7ZQA0QYF name_7ZQA institution
ENTITY OYYABW87 name_OYYA institution
ENTITY N22CM7B6 name_N22C paper
ENTITY 31BWXFUG name_31BW paper
ENTITY 7UDKGMP5 name_7UDK institution
ENTITY IZPI6FZZ name_IZPI institution
RELATIONSHIP SFLY3MWC collaborated_with LMWWZQST 6.0
RELATIONSHIP LMWWZQST affiliated_with 7ZQA0QYF 2.8
RELATIONSHIP LMWWZQST published_in 31BWXFUG 7.2
RELATIONSHIP 7UDKGMP5 collaborated_with UT3LHDM9 4.1
RELATIONSHIP SFLY3MWC cited_by N22CM7B6 1.9
RELATIONSHIP MRRX4FAW cited_by 7ZQA0QYF 7.6
RELATIONSHIP IZPI6FZZ affiliated_with N22CM7B6 4.4
RELATIONSHIP OYYABW87 presented_at 7UDKGMP5 10.0
RELATIONSHIP LMWWZQST affiliated_with OYYABW87 6.7
RELATIONSHIP MRRX4FAW presented_at N22CM7B6 1.1
RELATIONSHIP MRRX4FAW presented_at 31BWXFUG 1.9
RELATIONSHIP 7ZQA0QYF authored_by IZPI6FZZ 2.2
RELATIONSHIP N22CM7B6 affiliated_with UT3LHDM9 0.6
RELATIONSHIP MRRX4FAW affiliated_with SFLY3MWC 6.4
RELATIONSHIP 7ZQA0QYF cited_by 7UDKGMP5 3.1
PRINT 7UDKGMP5
HIGHEST
PATH MRRX4FAW LMWWZQST
HIGHEST
DELETE LMWWZQST
HIGHEST
PATH SFLY3MWC UT3LHDM9
HIGHEST
DELETE MRRX4FAW
DELETE SFLY3MWC
EXIT
