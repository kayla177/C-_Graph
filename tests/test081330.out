success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
UT3LHDM9 OYYABW87 7ZQA0QYF
UT3LHDM9 N22CM7B6 28.7
MRRX4FAW SFLY3MWC LMWWZQST 12.4
UT3LHDM9 N22CM7B6 28.7
success
SFLY3MWC OYYABW87 27.1
SFLY3MWC MRRX4FAW 7ZQA0QYF 7UDKGMP5 UT3LHDM9 21.2
SFLY3MWC OYYABW87 27.1
success
success
